js/store/reducers.js

```diff
@@ -32,12 +40,55 @@ function itemsFetching(state) {
 }
 
 function itemsFetched(state, { value }) {
+  var ids = [];
+  const toCache = value.reduce((result, item) => {
+    result[item.id] = createItem(FETCHED, item);
+    ids.push(item.id);
+    return result;
+  }, {});
```

Comment:
@samgiles I don't know enough about how this journey-data part will work yet. Are you saying Redux shouldn't hold any detailed item state? I was expecting the journey-data API would be treated similar to a REST API; Redux would fetch bunch of items from it, store them internally, pass them to React to render, and periodically update its 'cache'.